models/repo_pushmirror.go

```diff
@@ -57,6 +59,9 @@ func (m *PushMirror) GetRemoteName() string {
 
 // InsertPushMirror inserts a push-mirror to database
 func InsertPushMirror(m *PushMirror) error {
+	if setting.Mirror.DisableNewPush {
```

Comment:
I would like to move it out of the function and put it in router level.

Reply:
checked ... is in place